packages/contracts-watr/scripts/deployment/deployRegistry.ts

```diff
@@ -0,0 +1,16 @@
+import { contract, createProxy } from 'ethereum-mars'
+import { OwnedUpgradeabilityProxy, Registry } from '../../build/artifacts'
+
+export function deployRegistry() {
+  const ownedUpgradabilityProxy = createProxy(OwnedUpgradeabilityProxy)
+
+  const implementation = contract(Registry)
+  const proxy = ownedUpgradabilityProxy(implementation, (registry) => {
+    registry.initialize()
```

Comment:
We are not transfering `registryProxy` ownership to anyone, I guess the deployer shouldn't be the owner 🤔